.travis.yml

```diff
@@ -14,4 +14,16 @@ jdk:
   - oraclejdk8
 
 script:
-  - ./gradlew check --info -S --parallel
\ No newline at end of file
+  - ./gradlew check --info -S --parallel
+
+defore_deploy:
```

Comment:
Isn't this supposed to be `before_deploy`?

Reply:
Yes, it's typo.